wiki/FAQ/ja.md

```diff
@@ -3,18 +3,18 @@
 Osu!とは
 ========
 
-このゲームのメインとなるモードで、カーソルを動かしサークル、スライダー、スピナーの3つをタイミングよく叩くモードです
-叩くタイミングの正確さによって300、100、50、missと4つの判定が出ます
-また[太鼓モードや](/wiki/Game_Modes/osu!taiko)[CTBとは別々にスコア管理がされます](/wiki/Game_Modes/osu!catch)
+このゲームのメインとなるモードで、カーソルを動かしサークル、スライダー、スピナーの3つをタイミングよく叩くモードです。
+叩くタイミングの正確さによって300、100、50、missと4つの判定が出ます。
+また[太鼓モードや](/wiki/Game_Modes/osu!taiko)[CTBとは別々にスコア管理がされます](/wiki/Game_Modes/osu!catch)。
```

Comment:
```suggestion
また[太鼓モード](/wiki/Game_Modes/osu!taiko)や[osu!catch](/wiki/Game_Modes/osu!catch)とは別々にスコア管理がされます。
```
if there is a Japanese word for "osu!catch" you can use that instead

Reply:
I don't think game modes should be translated.

Reply:
I agree.
I'll change them.